### PR TITLE
Disable esModuleInterop setting - Closes #993

### LINKED ISSUE #933: Consider disabling esModuleInterop
### Description
Currently `esModuleInterop` is `true` at the root level of tsconfig.
As a library, this will limit TypeScript User to forcing them to use the same setting, too.

Therefore, it would be better if it's disabled.

### Version
2.0-

### PR DESCRIPTION
**Pull request by shuse2**
### What was the problem?
If esModuleInterop is true, there was no option to disable it on the client side.

### How did I fix it?
Turn off esModuleInterop on root level tsconfig, and fix the build and test error

### How to test it?
`npm t`

### Review checklist

* The PR resolves #933 
* All new code is covered with unit tests
* All new code was formatted with Prettier
* Linting passes
* Tests pass
* Commit messages follow the [commit guidelines](CONTRIBUTING.md#git-commit-messages)
* Documentation has been added/updated

### CODE REVIEW

File: packages/lisk-transactions/src/utils/validation/validator.ts [hunk 1]
```diff
@@ -12,9 +12,10 @@
  * Removal or modification of this copyright notice is prohibited.
  *
  */
-import Ajv from 'ajv';
-import addKeywords from 'ajv-merge-patch';
-import BigNum from 'browserify-bignum';
+import * as Ajv from 'ajv';
+// tslint:disable-next-line no-require-imports
+import addKeywords = require('ajv-merge-patch');
```
Comment (SargeKhan) [inline packages/lisk-transactions/src/utils/validation/validator.ts:17]: why do we need to use require here?

Comment (shuse2) [reply packages/lisk-transactions/src/utils/validation/validator.ts:17]: when library is exposed only function like `module.exports = function (xx) {}`, it cannot `import *` and not as default either

**Comment by shuse2** [comment]
Since we were using `--esModuleInterop ` and module as `common.js, `allowSyntheticDefaultImports` was true by default.

I think from the elements side, --esModuleInterop was working, but main reason is it enforces the users of elements to use the same setting

**Comment by SargeKhan** [comment]
@shuse2, I was thinking that if we set `--allowSyntheticDefaultImports` to true and `--esModuleInterop` to false then maybe we can use default imports from within the application without enforcing it for clients. (Since the documentation says that `--esModuleInterop` emit __importStar and __importDefault, these emits are not fired on `--allowSyntheticDefaultImports`.)

**Comment by shuse2** [comment]
@SargeKhan I see, I will try it out.
My first thought about `--allowSyntheticDefaultImports to true and --esModuleInterop to false `  was that will break the build file coz it does't have the Star or Default

**Comment by shuse2** [comment]
Suspended until new transaction module is merged back

**Pull request merged**